packages/dds/tree/src/test/utils.ts

```diff
@@ -1028,6 +1038,7 @@ export function applyTestDelta(
 	const rootDelta = rootFromDeltaFieldMap(delta, build, destroy);
 	applyDelta(
 		rootDelta,
+		undefined,
```

Comment:
While it's true that all current usages of `applyTestDelta` don't care what revision is passed in here, that may not be true of future usages. I'm concerned that some future test will try to use this function but not realize that some non-undefined revision need to be passed in. So, I would at least recommend that the revision be a parameter to this function (which perhaps* can default to undefined).

*I say "perhaps" because I'm interested in trying to remove all unnecessary usages of undefined for revisions. (In part so that we can use undefined when it really is necessary. For example, it would be nice if we could use undefined for `fakeRevisionWhenNotSet`)